index.bs

```diff
@@ -2216,6 +2218,46 @@ The primary difference between {{XRViewerPose}} and {{XRPose}} is the inclusion
 
 If the relationship between {{XRView}}s could uniquely identify the [=/XR device=], then the user agent MUST anonymized the {{XRView}} data to prevent fingerprinting. The method of anonymization is at the discretion of the user agent. Furthermore, if the relationship between {{XRView}}s is affected by a user-configured interpupillary distance, then it is strongly recommended that the user agent require [=explicit consent=] prior to reporting any {{XRView}} data.
 
+### Reference spaces ### {#protect-reference-spaces}
+Various reference space types have restrictions places on their use to prevent the unintentional exposure of [=sensitive information=].
+
+For any {{XRReferenceSpaceType}} included in an {{XRSession}}'s [=requested features=] the user agent MUST ensure the document is allowed to use all policy-controlled features associated with the sensor types used to track the [=native origin=] of the {{XRReferenceSpaceType}} or the feature MUST be rejected.
```

Comment:
The specific text from the explainer I'm pulling from here is:

> In response the UA must ensure that:
> * The document is allowed to use all the policy-controlled features associated with the sensor types used to track the native origin of an unbounded reference space

Which is much closer to the text in question. The text you point out is actually captured more explicitly in #761 (Specifically [this line](https://github.com/immersive-web/webxr/pull/761/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR2262)) which reads:

> In addition to the <code>"xr"</code> feature policy, feature policies for <dfn>underlying sensors</dfn> must also be respected if a site could isolate and extract sensor data that would otherwise be blocked by those feature policies. 

That's not to say that we can't or shouldn't make this text clearer if needed, I just want to make sure we're on the same page regarding where this text was pulled from.

Reply:
Good clarification, thanks... my questions still stand (though I should probably have asked them with relation to the explainer instead of now).

For me it boils down to whether the phrase "tracking the native origin" sufficiently covers the possibility that gyro data could be used for device orientation, including the viewer reference space. WDYT?